go.mod

```
@@ -3,15 +3,14 @@ module sigs.k8s.io/mcs-api
 go 1.14
 
 require (
-	github.com/go-logr/logr v0.3.0
-	github.com/go-logr/zapr v0.2.0 // indirect
-	github.com/onsi/ginkgo v1.16.2
-	github.com/onsi/gomega v1.11.0
-	golang.org/x/sys v0.0.0-20210503173754-0981d6026fa6 // indirect
-	k8s.io/api v0.19.0
-	k8s.io/apimachinery v0.19.0
-	k8s.io/client-go v0.19.0
-	sigs.k8s.io/controller-runtime v0.6.1
-	sigs.k8s.io/controller-tools v0.3.0
-	sigs.k8s.io/kind v0.8.1
+	github.com/go-logr/logr v0.4.0
+	github.com/onsi/ginkgo v1.16.4
+	github.com/onsi/gomega v1.16.0
+	k8s.io/api v0.22.2
+	k8s.io/apimachinery v0.22.2
+	k8s.io/client-go v0.22.2
+	k8s.io/code-generator v0.22.1
```

I guess all of the dependencies should consistent.
```suggestion
	k8s.io/code-generator v0.22.2
```